src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private static final Color[] colors = Color.values();
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
Let's just avoid static methods and variables for now(except for constant values like numbers) and reformat our solution by OOP style, which will be covered in detail later in the course.
Same for all other classes and their methods